### PR TITLE
libcec: create new sub package for python dependencies

### PR DESCRIPTION
**Pull request by Gottox**

### CODE REVIEW

File: srcpkgs/libcec/template [hunk 1]
```diff
@@ -39,6 +39,14 @@ cec-client_package() {
 		vmove usr/bin
 	}
 }
+python3-libcec_package() {
+	short_desc+=" - python bindings"
+	depends="python3 libcec>=${version}"
+
+	pkg_install() {
+		vmove usr/lib/python3.8
```
Comment (Chocimier) [inline srcpkgs/libcec/template:47]: There is futureproof `py3_lib` variable for that.

Comment (Gottox) [reply srcpkgs/libcec/template:47]: true. I could use that too, but the value of `py3_lib` starts with a `/` which is not recommended for vmove. Afaics nothing in the system relies on the starting slash. remove it from the value too?

Comment (Gottox) [reply srcpkgs/libcec/template]: fixed

Comment (Chocimier) [reply srcpkgs/libcec/template]: Right, values can drop slash then. #23073

**Pull request merged**